tools/who_what_benchmark/tests/test_cli_reranking.py

```diff
@@ -1,64 +1,94 @@
 import subprocess  # nosec B404
+import sys
 import pytest
+import shutil
 import logging
+import tempfile
 from test_cli_image import run_wwb
+from pathlib import Path
 
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
+tmp_dir = tempfile.mkdtemp()
 
 
-@pytest.mark.parametrize(
-    ("model_id", "model_type"),
-    [
-        ("cross-encoder/ms-marco-TinyBERT-L2-v2", "text-reranking"),
-    ],
-)
-def test_reranking_basic(model_id, model_type, tmp_path):
-    GT_FILE = tmp_path / "gt.csv"
-    MODEL_PATH = tmp_path / model_id.replace("/", "--")
-
-    result = subprocess.run(["optimum-cli", "export",
-                             "openvino", "-m", model_id,
-                             MODEL_PATH, "--task",
-                             "text-classification",
-                             "--trust-remote-code"],
-                            capture_output=True,
-                            text=True,
-                            )
-    assert result.returncode == 0
+OV_RERANK_MODELS = {
+    ("cross-encoder/ms-marco-TinyBERT-L2-v2", "text-classification"),
+    ("Qwen/Qwen3-Reranker-0.6B", "text-generation"),
+}
 
-    # Collect reference with HF model
+
+def setup_module():
+    for model_info in OV_RERANK_MODELS:
+        model_id = model_info[0]
+        task = model_info[1]
+        MODEL_PATH = Path(tmp_dir, model_id.replace("/", "_"))
+        subprocess.run(["optimum-cli", "export", "openvino", "--model", model_id, MODEL_PATH, "--task", task, "--trust-remote-code"],
+                       capture_output=True,
+                       text=True)
```

Comment:
If someone runs it locally with the intention to run a single test case, all of `OV_RERANK_MODELS` will be exported anyway. You should prefer fixtures. See samples tests. You can also reuse `OV_CACHE` env var and skip exporting if the model is already there.

But I don't think this should be addressed now. Seem to be a separate optional task.